apps/roam/src/components/DiscourseNodeSearchMenu.tsx

```diff
@@ -175,6 +184,8 @@ const NodeSearchMenu = ({
       .filter((type) => searchResults[type.type]?.length > 0);
   }, [discourseTypes, checkedTypes, searchResults]);
 
+  // derived selection set from checkedTypes; no separate selected list needed
```

Comment:
Is this describing `filteredTypes`?
Seems more accurate to say that it is derived from `discourseTypes`

Regardless, if you feel that i put it directly above variable and leave no spaces.
But only if it is adding additional context that the code itself doesn't make obvious.